frontend/.env.production.example

```
@@ -0,0 +1,11 @@
+# 本番環境用E2Eテスト設定
+JWT_SECRET_KEY=your-super-secure-256-bit-random-key
+TEST_USER_EMAIL=e2e-test@your-domain.com
+TEST_USER_PASSWORD=secure-test-password-123
+BASE_URL=https://your-production-domain.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BASE_URL key should go before the JWT_SECRET_KEY key

(UnorderedKey)